packages/cli/src/lib/cmds/help.ts

```diff
@@ -84,10 +86,29 @@ const HELP_MESSAGES = new Map<string, string>(
             '"./app-release-signed.apk"',
         '--verbose ................. prints the adb command being executed',
       ].join('\n')],
+      ['doctor', [
+        'Usage:',
+        '',
+        '',
+        'bubblewrap doctor',
+      ].join('\n')],
+      ['updateConfig', [
+        'Usage:',
+        '',
+        '',
+        // eslint-disable-next-line no-implicit-coercion
```

Comment:
What's this about?

Reply:
without that lint throws 'no-implicit-coercion' error and tells me to put the Number(<next-line>) for some reason

Reply:
That's weird - I wonder why it isn't needed for the other commands.

Reply:
There's a dangling `+` sign at the of the line below. Can that be the cause of the error? Generally, we want to avoid eslint-disable as much as possible.

Reply:
I'll check what can I do